docs/diagrams/FilterActivityDiagram.puml

```diff
@@ -3,19 +3,16 @@
 start
 :User executes filter command;
 switch ()
-case ( [keyword == "tags/"] )
-    :Filter pet list by tags given as search word;
-case ( [keyword == "date/"] )
-    :Filter pet list by date given as search word;
-case ( [keyword == "appointment/"] )
-    :Filter pet list by appointment date given as search word;
-case ( [keyword == "ownerName/"] )
-    :Filter pet list by owner name given as search word;
-case ( [keyword == "today"] )
-    :Filter pet list by today's attendance;
+case ( [field == "byTags/"] )
+    :Filter pet list by tags given as filter word;
+case ( [field == "byDate/"] )
+    :Filter pet list by date given as filter word;
```

Comment:
I noticed this too but I was not able to fix it somehow on friday, will try again

Reply:
I have edited this since we can filter by 4 different fields now. 
![FilterActivityDiagram](https://user-images.githubusercontent.com/77776878/160283191-ef2857ac-26d2-4951-96a8-da3f80b04bab.png)